sdk/go/common/workspace/creds.go

```diff
@@ -39,6 +39,8 @@ const PulumiCredentialsPathEnvVar = "PULUMI_CREDENTIALS_PATH"
 // used instead of the currently logged in backend or the current projects backend.
 const PulumiBackendURLEnvVar = "PULUMI_BACKEND_URL"
 
+const LoadCurrentProjectFailMsg = "could not load current project"
```

Comment:
Could this be a structured error that we could use with `errors.Is` instead of a string comparison?